src/js/component/search.vue

```diff
@@ -1,16 +1,17 @@
 <template>
-    <section class="search" v-if="currentQuery !== null">
+    <section class="search">
         <div class="search__label">Results&nbsp;for</div>
         <div class="search__query">{{ currentQuery }}</div>
     </section>
 </template>
 
 <script>
-    import { mapGetters } from 'vuex'
```

Comment:
Same here, I prefer to get read of the store as a global dependency. Before you know it, it's everywhere.
Local state for component is great is that it is scoped, global state is not good for everything.